packages/experience-builder-sdk/src/index.ts

```diff
@@ -17,8 +17,6 @@ export {
   CF_STYLE_ATTRIBUTES,
 } from '@contentful/experiences-core/constants';
 
-export { detachExperienceStyles } from './utils/ssrStyles';
```

Comment:
to keep the public API untouched, we should keep the export here as the following:
```
export { detachExperienceStyles } from '@contentful/experiences-core'
```

Otherwise, all our code examples will be affected and all our existing customers, who imported this fn from our react-sdk package would get an error saying that there is no such export